tests/e2e/pageObjects/browser-page.ts

```diff
@@ -892,3 +912,13 @@ export type ListKeyParameters = {
     keyName: string,
     element: string
 };
+
+/**
+ * Keys Data parameters
+ * @param textType The type of the key
+ * @param keyName The name of the key
+ */
+export type keyData = {
```

Comment:
Where do you use this type? Also, in type names, we use PascalCase

Reply:
Thanks, fixed!
I use it in addKeysViaCli and deleteKeysViaCli functions as parameter type